baseband/vdif/base.py

```diff
@@ -277,29 +277,29 @@ def __repr__(self):
 class VDIFStreamReader(VDIFStreamBase, VLBIStreamReaderBase, VDIFFileReader):
     """VLBI VDIF format reader.
 
-    This wrapper allows one to access a VDIF file as a continues series of
-    samples.  Invalid data are marked, but possible gaps in the data stream
-    are not yet filled in.
+    Allows access to a VDIF file as a continuous series of samples.  Invalid
+    data are marked, but possible gaps in the data stream are not yet filled
```

Comment:
As for Mark 5B, maybe just remove what we do not yet do. In fact, even remove the invalid part? That will soon be covered by `fill_value` anyway.